capstone-frontend/src/App.js

```diff
@@ -1,13 +1,27 @@
+
 import React from 'react';
+import logo from './logo.svg';
 import './App.css';
-import { TestApi } from "./components/TestApi";
 
 function App() {
   return (
-    <div id="testApiID">
-      <TestApi />
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
     </div>
```

Comment:
nit: This can probably be replaced with something generic, like a header that just says "BookBook" even